robottelo/datafactory.py

```diff
@@ -179,9 +179,9 @@ def valid_emails_list():
             gen_string('alphanumeric'),
             gen_string('alphanumeric'),
         ),
-        u'"():;"@example.com',
```

Comment:
Updated the unit test, thanks. I removed that because it was breaking on a cli test, another case that needed escaping, this time the brackets